packages/react/src/publication/useNotInterestedToggle.ts

```diff
@@ -37,7 +35,7 @@ export type NotInterestedOperation = UseDeferredTask<void, never, UseNotInterest
  * @category Publications
  * @group Hooks
  */
-export function useNotInterestedToggle(): NotInterestedOperation {
+export function useNotInterestedToggle(): UseDeferredTask<void, never, UseNotInterestedToggleArgs> {
```

Comment:
No objection per se, just curious about the rationale for removing this alias.

Reply:
I just found them unnecessary, and not consistent with other hooks, it's a breaking change though

Reply:
Understood. I wouldn't worry too much about a breaking change. I think it's very unlikely anybody referenced these types in code.